docs/src/introduction/getting-started.md

```diff
@@ -1 +1,131 @@
-> [WIP]
+# Getting started
+
+This tutorial guides you through the process of connecting to a remote Miden-Node using the Miden client. The Miden node; that is, the software which processes transactions and creates blocks for the Miden rollup and the Miden client, which provides a way to execute and prove transactions, facilitating the interaction with the Miden rollup. By the end of this tutorial, you will be able to configure the Miden client, connect to a Miden-Node, and perform basic operations like sending transactions, generating and consuming notes.
+
+## Prerequisites
+
+Before starting, ensure you have the following:
+
+- **Rust Installed:** You must have the Rust programming language installed on your machine. If you haven't installed Rust, you can download it from [the Rust website](https://www.rust-lang.org/learn/get-started).
+- **Node IP Address:** Obtain the IP address of the running Miden-Node. This information can be acquired by contacting one of the Miden engineers.
+- **Miden-Client Installation:** You need to install the [Miden client](https://github.com/0xPolygonMiden/miden-client) and configure it to point to the remote node.
+
+## Step 1: Configuring the Miden-Client
+
+1. **Download the Miden-Client:** First, download the Miden client from its repository. Use the following command:
+
+   ```shell
+   git clone https://github.com/0xPolygonMiden/miden-client
+   ```
+
+2. **Navigate & Configure the Client:** Navigate to the client directory and modify the configuration file to point to the remote Miden-Node. You can find the configuration file at `./miden.toml`. In the `[RPC]` section replace the `endpoint = { host: }` field with the address provided by the Miden engineer.
+
+   ```shell
+   cd miden-client
+   ```
+
+   Configuration file example:
+
+   ```toml
+    [rpc]
+    endpoint = { protocol = "http", host = "<NODE_IP_ADDRESS>", port = 57291 }
+
+    [store]
+    database_filepath = "store.sqlite3"
+   ```
+
+3. **Build & install the Client:** install the client using cargo:
+
+   ```shell
+   cargo install --features testing --path .
```

Comment:
Can you add `--features testing,concurrent`? 

I think there is no reason not to run in concurrent, right? We can add a line after this command and say `if concurrent does not work due to xyz, just use --features testing`